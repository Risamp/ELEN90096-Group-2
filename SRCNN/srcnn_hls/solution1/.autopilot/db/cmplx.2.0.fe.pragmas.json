{
  "0": {
    "basicblocks": 1,
    "calls": "3 1 4",
    "file": "srcnn.cpp",
    "fileloc": 4,
    "instructions": 4,
    "name": "srcnn",
    "total_instructions": 519
  },
  "1": {
    "basicblocks": 17,
    "calls": "2 2",
    "file": "conv2.cpp",
    "fileloc": 8,
    "instructions": 78,
    "loads": {
      "G": 3,
      "L": 2
    },
    "name": "conv2",
    "stores": {
      "G": 3
    },
    "total_instructions": 96
  },
  "2": {
    "basicblocks": 6,
    "calls": "",
    "file": "util.cpp",
    "fileloc": 82,
    "instructions": 9,
    "name": "clamp",
    "total_instructions": 9
  },
  "3": {
    "basicblocks": 42,
    "calls": "2 2 2 2 2 2",
    "file": "conv1.cpp",
    "fileloc": 9,
    "instructions": 275,
    "loads": {
      "G": 9,
      "L": 9
    },
    "name": "conv1",
    "stores": {
      "G": 9
    },
    "total_instructions": 329
  },
  "4": {
    "basicblocks": 15,
    "calls": "2 2",
    "file": "conv3.cpp",
    "fileloc": 8,
    "instructions": 72,
    "loads": {
      "G": 1,
      "L": 4
    },
    "name": "conv3",
    "stores": {
      "L": 2
    },
    "total_instructions": 90
  }
}