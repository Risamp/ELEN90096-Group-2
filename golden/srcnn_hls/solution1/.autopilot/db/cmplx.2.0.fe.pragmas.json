{
  "0": {
    "basicblocks": 1,
    "calls": "2 4 1",
    "file": "srcnn.cpp",
    "fileloc": 4,
    "instructions": 4,
    "name": "srcnn",
    "total_instructions": 288
  },
  "1": {
    "basicblocks": 15,
    "calls": "3 3",
    "file": "conv3.cpp",
    "fileloc": 8,
    "instructions": 72,
    "loads": {
      "G": 1,
      "L": 4
    },
    "name": "conv3",
    "stores": {
      "L": 2
    },
    "total_instructions": 90
  },
  "2": {
    "basicblocks": 17,
    "calls": "3 3",
    "file": "conv1.cpp",
    "fileloc": 9,
    "instructions": 80,
    "loads": {
      "G": 2,
      "L": 3
    },
    "name": "conv1",
    "stores": {
      "G": 3
    },
    "total_instructions": 98
  },
  "3": {
    "basicblocks": 6,
    "calls": "",
    "file": "util.cpp",
    "fileloc": 82,
    "instructions": 9,
    "name": "clamp",
    "total_instructions": 9
  },
  "4": {
    "basicblocks": 17,
    "calls": "3 3",
    "file": "conv2.cpp",
    "fileloc": 8,
    "instructions": 78,
    "loads": {
      "G": 3,
      "L": 2
    },
    "name": "conv2",
    "stores": {
      "G": 3
    },
    "total_instructions": 96
  }
}